ios/Fabric/RNCSafeAreaViewComponentView.mm

```diff
@@ -47,11 +47,34 @@ - (NSString *)description
   if (superDescription.length > 0 && [superDescription characterAtIndex:superDescription.length - 1] == '>') {
     superDescription = [superDescription substringToIndex:superDescription.length - 1];
   }
+  
+#if TARGET_OS_IPHONE
+  NSString *providerViewSafeAreaInsetsString = NSStringFromUIEdgeInsets(_providerView.safeAreaInsets);
+  NSString *currentSafeAreaInsetsString = NSStringFromUIEdgeInsets(_currentSafeAreaInsets);
+#elif TARGET_OS_OSX
+  NSString *providerViewSafeAreaInsetsString;
+  NSString *currentSafeAreaInsetsString;
+  if (@available(macOS 11.0, *)) {
```

Comment:
Does RN actually support this target?

Reply:
This is required if we want to support 0.76, from 0.77 onwards the minimum required version for macOS is 11.0. Should I remove this check?

Reply:
We only support new arch for the latest versions of RN, so sounds like we could just remove this - or remove it to the extent the compiler will allow